docs/index.asciidoc

```diff
@@ -41,24 +41,44 @@ Example:
 ----------------------------------
 input {
   exec {
-    command => "ls"
+    command => "echo 'hi!'"
     interval => 30
   }
 }
 ----------------------------------
 
-This will execute `ls` command every 30 seconds.
+This will execute `echo` command every 30 seconds.
+
+[id="plugins-{type}s-{plugin}-ecs"]
+==== Compatibility with the Elastic Common Schema (ECS)
+
+This plugin adds metadata about the event's source, and can be configured to do so
+in an {ecs-ref}[ECS-compatible] way with <<plugins-{type}s-{plugin}-ecs_compatibility>>.
+This metadata is added after the event has been decoded by the appropriate codec,
+and will not overwrite existing values.
+
+|========
+| ECS Disabled | ECS v1 , v8| Description
+
+| `host`  | `[host][name]` | The name of the {ls} host that processed the event
+| `command` | `[process][command_line]` | The command run by the plugin
+| `[@metadata][exit_status]` | `[process][exit_code]` | The exit code of the process
+
+| `[@metadata][input][exec][process][elapsed_time]` | -- | The elapsed time the command took to run in nanoseconds
```

Comment:
Implementation says this is [the opposite](https://github.com/logstash-plugins/logstash-input-exec/pull/28/files#diff-fdfe5b5dd3ccef8cc9228830f8800cc2bbfdd9f84ca59d15010357409dee7ed9R48)

```suggestion
| -- | `[@metadata][input][exec][process][elapsed_time]` | The elapsed time the command took to run in nanoseconds
```